base/loading.jl

```diff
@@ -408,9 +408,33 @@ end
 ## generic project & manifest API ##
 
 const project_names = ("JuliaProject.toml", "Project.toml")
-const manifest_names = ("JuliaManifest.toml", "Manifest.toml")
+const _manifest_names = ("JuliaManifest.toml", "Manifest.toml")
 const preferences_names = ("JuliaLocalPreferences.toml", "LocalPreferences.toml")
 
+function manifest_names()
+    custom_def = get(ENV, "JULIA_MANIFEST_NAME", nothing)
+    if !isnothing(custom_def) && !isempty(custom_def)
```

Comment:
```suggestion
    if !isempty(custom_def)
```

Reply:
I believe this suggestion is wrong. `isempty(nothing)` is an error

Reply:
But we now have `custom_def = strip(get(ENV, env_var_name, ""))`, right? So `custom_def` should always be an `AbstractString`.